services/auth/authUser.js

```diff
@@ -0,0 +1,48 @@
+const { jwtSign } = require('../../handlers');
+
+// our user "table"
+const users = [
+  {
+    useremail: 'someone@nowhere.com',
```

Comment:
I think it makes sense to just name this property `email` so it can be accessed as `user.email`